### ISSUE TITLE
fabric-ccenv-sgx build is broken

### ISSUE THREAD

**Issue by mbrandenburger** [hyperledger-labs/fabric-private-chaincode#163]
When building fabric-ccenv-sgx docker image I get the following error:

```
--2019-09-18 17:59:20--  http://registrationcenter-download.intel.com/akdlm/irc_nas/11414/iclsClient-1.45.449.12-1.x86_64.rpm
Resolving registrationcenter-download.intel.com (registrationcenter-download.intel.com)... 23.54.112.14, 2a02:26f0:3000:29e::b, 2a02:26f0:3000:28c::b
Connecting to registrationcenter-download.intel.com (registrationcenter-download.intel.com)|23.54.112.14|:80... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: /en/irc_nas/11414/iclsclient-1.45.449.12-1.x86_64.rpm/ [following]
--2019-09-18 17:59:21--  http://registrationcenter-download.intel.com/en/irc_nas/11414/iclsclient-1.45.449.12-1.x86_64.rpm/
Reusing existing connection to registrationcenter-download.intel.com:80.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://www-ssl.intel.com/content/www/us/en/404.html?aspxerrorpath=/en/irc_nas/11414/iclsclient-1.45.449.12-1.x86_64.rpm/ [following]
--2019-09-18 17:59:21--  https://www-ssl.intel.com/content/www/us/en/404.html?aspxerrorpath=/en/irc_nas/11414/iclsclient-1.45.449.12-1.x86_64.rpm/
Resolving www-ssl.intel.com (www-ssl.intel.com)... 23.54.112.14
Connecting to www-ssl.intel.com (www-ssl.intel.com)|23.54.112.14|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-09-18 17:59:21 ERROR 404: Not Found.
```

seems that Intel has decided to remove iclsclient from their servers ....

**Comment by bvavala** [comment]
I cannot reproduce it -- it seems to be working.

**Comment by g2flyer** [comment]
> I cannot reproduce it -- it seems to be working.

It works because docker uses the cached version built earlier.  It will fail if you remove docker cache (`docker system prune`) or run as `docker build --no-cache -t hyperledger/fabric-ccenv-sgx:latest docker-sgx` .  Interesting that we notice only now because i thought the iclsclient disappeared a while ago ...

**Comment by mbrandenburger** [comment]
mh ... working on an alternative

https://github.com/mbrandenburger/fabric-secure-chaincode/tree/new-docke-base

**Comment by mbrandenburger** [comment]
also ... upgrading to 2.6 will fix the issue I have with Bruno's current PR #158

**Comment by mbrandenburger** [comment]
PR #168  addresses this issue and updates to SGX SDK 2.6

**Comment by mbrandenburger** [comment]
Fixed with #168
